components/sync_device_info/brave_device_info.cc

```diff
@@ -96,6 +96,12 @@ std::string BraveDeviceInfo::GetDeviceTypeString() const {
       return "phone";
     case FormFactor::kTablet:
       return "tablet";
+    case FormFactor::kAutomotive:
```

Comment:
This string is used to display device icon at the device list. As we don't have automotive/wearable/tv - this is ok to say tablet.